docs/apis/streaming/connectors/kafka.md

```diff
@@ -142,18 +142,24 @@ for querying the list of topics and partitions.
 For this to work, the consumer needs to be able to access the consumers from the machine submitting the job to the Flink cluster.
 If you experience any issues with the Kafka consumer on the client side, the client log might contain information about failed requests, etc.
 
-##### The `DeserializationSchema`
+##### **The `DeserializationSchema`**
```

Comment:
The whole doc uses pretty low weight formatting. It is hard to recognize a heading as such. I overlooked the section initially.